jest-common/src/main/java/io/searchbox/core/Cat.java

```diff
@@ -0,0 +1,92 @@
+package io.searchbox.core;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import io.searchbox.action.AbstractAction;
+import io.searchbox.action.AbstractMultiIndexActionBuilder;
+import io.searchbox.action.AbstractMultiTypeActionBuilder;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+/**
+ * @author Bartosz Polnik
+ */
+public class Cat extends AbstractAction<CatResult> {
+    private final static String PATH_TO_RESULT = "result";
+    public Cat(AbstractAction.Builder<Cat, ? extends Builder> builder) {
+        super(builder);
+    }
+
+    Cat() {
+    }
+
+    @Override
+    public String getRestMethodName() {
+        return "GET";
+    }
+
+    @Override
+    public String getPathToResult() {
+        return PATH_TO_RESULT;
+    }
+
+    @Override
+    public CatResult createNewElasticSearchResult(String responseBody, int statusCode, String reasonPhrase, Gson gson) {
+        return createNewElasticSearchResult(new CatResult(gson), responseBody, statusCode, reasonPhrase, gson);
+    }
+
+    @Override
+    protected JsonObject parseResponseBody(String responseBody) {
+        JsonObject result = new JsonObject();
+        if (responseBody != null && !responseBody.trim().isEmpty()) {
+            result.add(PATH_TO_RESULT, new JsonParser().parse(responseBody).getAsJsonArray());
+        }
+
+        return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (obj == null) {
+            return false;
+        }
+        if (obj == this) {
+            return true;
+        }
+        if (obj.getClass() != getClass()) {
+            return false;
+        }
+
+        return new EqualsBuilder()
+                .appendSuper(super.equals(obj))
+                .isEquals();
+    }
+
+    public static class IndicesBuilder extends AbstractMultiTypeActionBuilder<Cat, IndicesBuilder> {
+        public IndicesBuilder() {
+            setHeader("content-type", "application/json");
```

Comment:
(i) getPlainText returns String[][]  which can be returned from controller in webapp and it will be neatly formatted - I consider this human readable
(ii) I don't know whether we can get the same info with different actions

There are also other benefits:
1. we can customize cat action, so that only a subset of columns will be returned (setParameter on builders)
2. we can reorder columns in final response
3. String[][] is easier to manipulate than json, when this is what we need
When creating cat action I wanted to give also these features.

I considered not adding application/json header to request, but then it would be difficult to integrate cat response with JestResult - (i.e. implementation of getJsonObject could be misleading).